articles/ros2_node_idl/setup.py

```diff
@@ -0,0 +1,19 @@
+TDB
+
+import glob
+import os
+
+from setuptools import setup
+
+package_name = "foo_py"
+
+setup(
+    name=package_name,
+    data_files=[
+        ('share/' + package_name, ['package.xml']),
+        ('share/ament_index/resource_index/packages',
+            ['resource/' + package_name]),
+        ('share/' + package_name + '/xml_cache', glob.glob('xml_cache/**')),
```

Comment:
@Arnatious did you already investigate supporting multiple XML files?

Reply:
Currently with `ament_nodl`, multiple export calls should correctly install multiple files. The parser can get a list of all nodl files from the marker file (line delimited)

Reply:
Okay so in this example, the use of xml_cache doesn't impact anything-- as long as the marker file includes those paths? I don't see that particular marker being installed here though; it probably should be. I think we can simplify this example to only be one file, too, thoughts?